lib/plugins/aws/deploy/compile/events/apiGateway/tests/apiKeys.js

```diff
@@ -38,23 +38,12 @@ describe('#compileApiKeys()', () => {
       region: 'us-east-1',
     };
     awsCompileApigEvents = new AwsCompileApigEvents(serverless, options);
-    awsCompileApigEvents.serverless.service.functions = {
-      first: {
-        events: [
-          {
-            http: {
-              path: 'users/create',
-              method: 'POST',
-              private: true,
-            },
-          },
-        ],
-      },
-    };
+    awsCompileApigEvents.restApiLogicalId = 'ApiGatewayRestApi';
+    awsCompileApigEvents.apiGateWayDeploymentLogicalId = 'ApiGatewayDeploymentTest';
```

Comment:
Could you rename this from `apiGateWayDeploymentLogicalId` to `apiGatewayDeploymentLogicalId` so that we have an overall consistent naming schema?

Thanks 👍